recipes/libcurl/all/conanfile.py

```diff
@@ -207,13 +204,22 @@ def build_requirements(self):
         else:
             self.build_requires("libtool/2.4.6")
             self.build_requires("pkgconf/1.7.4")
-            if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            if self._settings_build.os == "Windows" and not get_env("CONAN_BASH_PATH"):
                 self.build_requires("msys2/cci.latest")
```

Comment:
```suggestion
                self.tool_requires("msys2/cci.latest")
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#requirements